### PR TITLE
added endpoints for faulty sensor and connection to the mongodb

### PR DESCRIPTION
**Pull request by oererenn**

### CODE REVIEW

**Review by MolotovHammr — COMMENTED** [review]
Looks great, good job!

**Review by michaelbahchevanov — CHANGES_REQUESTED** [review]
Could you add tests as well?

**Comment by BondarencoM** [comment]
@oererenn What's weird is that you depend on mongoose, but don't have it in your list of dependencies in package.json. Obviously for some reason you have it locally.. Do you know what's going on?

**Review by larsverp — DISMISSED** [review]

**Review by MolotovHammr — DISMISSED** [review]

**Review by MolotovHammr — DISMISSED** [review]

File: GraphQL_Express_Server/globalConfig.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"mongoUri":"mongodb://127.0.0.1:60641/c592808e-5022-4bab-b980-98364e159776?"}
```
Comment (BondarencoM) [inline GraphQL_Express_Server/globalConfig.json]: What is the motivation for this file?

File: GraphQL_Express_Server/graphql/resolvers.js [hunk 2]
```diff
@@ -1,5 +1,20 @@
+const { SensorFault } = require("../models/sensorfault");
+
 module.exports = {
     Query: {
         MeanClimateMeasurements: (_, {start, stop}, { dataSources }) => dataSources.influx.getMeanClimateValues(start, stop),
+        SensorFaults: () => SensorFault.find({
```
Comment (BondarencoM) [inline GraphQL_Express_Server/graphql/resolvers.js]: Do you think we could add a datasaource object and call a method from that object when the resolver is invoked?

File: GraphQL_Express_Server/models/sensorfault.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+const mongoose = require("mongoose");
+
+const sensorFaultSchema = new mongoose.Schema({
+    loc_x: Number,
+    loc_y: Number,
+    floor: Number,
+    timestamp: {
+        type: Date,
+        default: Date.now - 1000 * 60 * 60
```
Comment (BondarencoM) [inline GraphQL_Express_Server/models/sensorfault.js]: Why is the default timestamp in the past?

File: GraphQL_Express_Server/test/mongodbConnection.test.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    await mongoose.connection.close()
+})
+
+beforeEach(async () => {
+            for (const key in mongoose.connection.collections) {
+                await mongoose.connection.collections[key].deleteMany();
+            }
+        })
+
+test('test of reading the entry', async () => {
```
Comment (BondarencoM) [inline GraphQL_Express_Server/test/mongodbConnection.test.js]: We need to also test that the time range works as intended.

**Review by BondarencoM — CHANGES_REQUESTED** [review]

**Review by BondarencoM — APPROVED** [review]

**Review by michaelbahchevanov — APPROVED** [review]
Looks great!

**Review by MolotovHammr — APPROVED** [review]

**Review by larsverp — APPROVED** [review]
rip package-lock

**Pull request merged**